src/lib/pages/schedules.svelte

```diff
@@ -52,53 +94,69 @@
     >
 
     <div class="flex flex-col gap-4" slot="header" let:visibleItems>
+      {@const showActions = visibleItems.length || query}
       <h1 class="flex flex-col gap-0 md:flex-row md:items-center md:gap-2">
         <SchedulesCount />
       </h1>
-
-      <div class="flex flex-row flex-wrap justify-between gap-2">
-        {#if visibleItems.length}
-          <div class="w-96">
-            <Input
-              icon="search"
-              type="search"
-              label={translate('schedules.name')}
-              labelHidden
-              id="schedule-name-filter"
-              placeholder={translate('schedules.name')}
-              clearable
-              clearButtonLabel={translate('common.clear-input-button-label')}
-              bind:value={search}
-              on:submit={noop}
-            />
-          </div>
-        {/if}
-        {#if !createDisabled && visibleItems.length}
+      {#if showActions}
+        <div class="w-96">
+          <Input
```

Comment:
Wondering if we want to keep this client side search for now or just remove it? Filtering by Schedule Id should be available with the next release.

Reply:
[Added](https://github.com/temporalio/ui/pull/2272/commits/2228c806b9f118422c93f23b90b878cc0ec291d1) ✔️